extension/observer/dockerobserver/README.md

```diff
@@ -0,0 +1,67 @@
+# Docker Observer Extension
+
+**Status: beta**
```

Comment:
The status should probably indicate that it's just a skeleton and cannot be used right now.